### PR TITLE
doing the lambda thing correctly

### PR DESCRIPTION
**Pull request by brentpicasso**
Correctly use lambda to schedule the status update on the next UI tick.

### CODE REVIEW

File: autosportlabs/racecapture/views/status/statusview.py [hunk 1]
```diff
@@ -163,7 +163,10 @@ def _on_menu_select(self, instance, value):
         self.update()
 
     def _on_status_updated(self, status):
-        self.status = status['status']
+        def safe_update_status(status):
+            self.status = status['status']
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/status/statusview.py]: Is this how we've normally done this, a nested function? I swear I've seen us use addListener and pass a lambda. I'm mostly asking so I know how to do this properly in the future.

**Comment by brentpicasso** [comment]
I just looked how the databus works and it's a completely disconnected process. with the Clock.schedule(). 

in main.py we have :

  def on_read_config_complete(self, rcpCfg):
        Clock.schedule_once(lambda dt: self.configView.dispatch('on_config_updated', self.rc_config))
        self.rc_config.stale = False
        self.showActivity('')

looking more..

**Comment by ryandoherty** [comment]
For the config example, I think the status stuff is handled differently because the status attribute is a Kivy Property. That may be why things are different.

**Comment by brentpicasso** [comment]
trying to see if it can be put in the .addListener call as was originally tried.

**Comment by brentpicasso** [comment]
the current way is the cleanest I can do at the moment.

**Comment by ryandoherty** [comment]
:shipit:

**Pull request merged**